src/main/resources/swagger/api-docs.yaml

```diff
@@ -71,6 +71,12 @@ paths:
             labels as top-level query string parameters. For instance: GET /api/clusters?key1=val1&key2=val2.
           required: false
           type: string
+        - in: query
+          name: includeDeleted
+          description: ugh
```

Comment:
heh

Reply:
hey that's my reaction too when working with swagger!